CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
+## 0.10.8
```

Comment:
Shouldn't this be 0.10.7+1?

Reply:
Wasn't planning to release – so keeping at `0.10.8` to align w/ the pubspec version.

Reply:
That's not the usual pattern. The CHANGELOG should reflect the actual version that this would be released as if it were released; the only reason the pubspec doesn't is that it's impossible to represent there.